### DISCUSSION TITLE
Deploy a SSG to Netlify

### DISCUSSION THREAD

**Discussion by Darksoulsong** [vercel/next.js#14332] [Help] [answered]
My web app was building alright and being exported correctly, then I decided to go from pure static to SSG by adding `fallback: true` to some of my blog pages. 

SSGs cannot be exported, so there isn't an "out/" folder with an index file that can be served. Netlify builds as it should, but all I see when I go ahead and navigate to the preview URL is a 404 page. 

The only way I can think to sort this out is to point to the actual build folder, so Netlify can serve my web app correctly. Where is the build folder save after build? Any help would be great.

**Comment by jamesmosier** [comment, marked as answer]
Netlify cannot handle dynamic sites, only static sites. Normally you'd run `next start` to start up your dynamic Next site (which is a Node server at its core) but Netlify cannot handle that.

They do have [serverless function](https://www.netlify.com/products/functions/) capabilities that might work, but would require some additional setup.

**Reply by Darksoulsong** [reply]
@jamesmosier Thank you for the reply. I think I'll stick with pure static for now.

**Reply by timneutkens** [reply]
If you want to take advantage of all Next.js features I'd recommend checking out vercel.com: https://nextjs.org/docs/deployment#vercel-recommended
